src/Converter/Extensions/InstagramConverter.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Predmond\HtmlToAmp\Converter\Extensions;
+
+use League\Event\EventInterface;
+use Predmond\HtmlToAmp\ElementInterface;
+use Predmond\HtmlToAmp\Converter\ConverterInterface;
+
+class InstagramConverter implements ConverterInterface
+{
+    public function hasInstagram(ElementInterface $element)
+    {
+        $classes = explode(' ', $element->getAttribute('class'));
+        return in_array('instagram-media', $classes);
+    }
+
+    public function findTheCode($element) 
+    {
+        $href = $element->getAttribute('href');
```

Comment:
As mentioned lower, I prefer $this->getEmbedId($element) or $this->getEmbedShortcode($element) for this method name.